Content.Shared/Clothing/EntitySystems/FireProtectionSystem.cs

```diff
@@ -9,15 +12,39 @@ namespace Content.Shared.Clothing.EntitySystems;
 /// </summary>
 public sealed class FireProtectionSystem : EntitySystem
 {
+    [Dependency] private readonly ExamineSystemShared _examine = default!; // WD
+
     public override void Initialize()
     {
         base.Initialize();
 
         SubscribeLocalEvent<FireProtectionComponent, InventoryRelayedEvent<GetFireProtectionEvent>>(OnGetProtection);
+        SubscribeLocalEvent<FireProtectionComponent, GetVerbsEvent<ExamineVerb>>(OnProtectionVerbExamine); // WD
     }
 
     private void OnGetProtection(Entity<FireProtectionComponent> ent, ref InventoryRelayedEvent<GetFireProtectionEvent> args)
     {
         args.Args.Reduce(ent.Comp.Reduction);
     }
+
+    // WD EDIT START
+    private void OnProtectionVerbExamine(EntityUid uid, FireProtectionComponent component, GetVerbsEvent<ExamineVerb> args)
```

Comment:
```suggestion
    private void OnProtectionVerbExamine(Entity<FireProtectionComponent> entity, ref GetVerbsEvent<ExamineVerb> args)
```

Используй подписки нового образца через Entity<>. Компонент получать через entity.Comp.хуй. А сам Ентити везде может спокойно выступать в роли EntityUid